+ source ./common/disk_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment!'
+ message='Start to prepare the test environment!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment!'
Thu Aug 31 17:28:34 2023 - INFO  - Start to prepare the test environment!
+ DNF_INSTALL lvm2
+ pkgs=lvm2
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs lvm2 --node 1 --tempfile ''
Thu Aug 31 17:28:42 2023 - INFO  - pkgs:(lvm2) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ check_free_disk
++ TEST_DISK
++ id=1
++ python3 /root/mugen-riscv/libs/locallibs/get_test_device.py --device disk --node 1
+ disks=
+ disk_list=($disks)
+ local_disk=
+ local_disk1=
+ local_disk2=
+ local_disk3=
+ LOG_INFO 'End to prepare the test environment!'
+ message='End to prepare the test environment!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment!'
Thu Aug 31 17:28:51 2023 - INFO  - End to prepare the test environment!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase!'
+ message='Start executing testcase!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase!'
Thu Aug 31 17:28:51 2023 - INFO  - Start executing testcase!
+ pvcreate -y /dev/
  No device found for /dev/.
+ CHECK_RESULT 5 0 0 'create PV failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='create PV failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'create PV failed'
+ LOG_ERROR 'create PV failed'
+ message='create PV failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'create PV failed'
Thu Aug 31 17:28:52 2023 - ERROR - create PV failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 30'
+ message='oe_test_lvm2_vgchange_002.sh line 30'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 30'
Thu Aug 31 17:28:53 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 30
+ return 0
+ pvs
+ grep /dev/
+ CHECK_RESULT 1 0 0 'create PV failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='create PV failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'create PV failed'
+ LOG_ERROR 'create PV failed'
+ message='create PV failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'create PV failed'
Thu Aug 31 17:28:53 2023 - ERROR - create PV failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 32'
+ message='oe_test_lvm2_vgchange_002.sh line 32'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 32'
Thu Aug 31 17:28:54 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 32
+ return 0
+ vgcreate test /dev/
  No device found for /dev/.
+ CHECK_RESULT 5 0 0 'create VG failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='create VG failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'create VG failed'
+ LOG_ERROR 'create VG failed'
+ message='create VG failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'create VG failed'
Thu Aug 31 17:28:55 2023 - ERROR - create VG failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 34'
+ message='oe_test_lvm2_vgchange_002.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 34'
Thu Aug 31 17:28:55 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 34
+ return 0
+ vgdisplay
+ grep 'VG Name'
+ grep test
+ CHECK_RESULT 1 0 0 'create VG failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='create VG failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'create VG failed'
+ LOG_ERROR 'create VG failed'
+ message='create VG failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'create VG failed'
Thu Aug 31 17:28:56 2023 - ERROR - create VG failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 36'
+ message='oe_test_lvm2_vgchange_002.sh line 36'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 36'
Thu Aug 31 17:28:57 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 36
+ return 0
+ vgchange --longhelp
+ grep 'Change volume group attributes'
  vgchange - Change volume group attributes
+ CHECK_RESULT 0 0 0 'vgchange --longhelp failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='vgchange --longhelp failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ vgchange --help
+ grep 'Change volume group attributes'
  vgchange - Change volume group attributes
+ CHECK_RESULT 0 0 0 'vgchange --help failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='vgchange --help failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ vgchange --version
+ grep 'LVM version'
  LVM version:     2.03.21(2) (2023-04-21)
+ CHECK_RESULT 0 0 0 'failed to test vgchange --version'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='failed to test vgchange --version'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ vgchange --poll y --ignorelockingfailure test
  Volume group "test" not found
  Cannot process volume group test
+ CHECK_RESULT 5 0 0 'failed to test vgchange --poll y'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='failed to test vgchange --poll y'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'failed to test vgchange --poll y'
+ LOG_ERROR 'failed to test vgchange --poll y'
+ message='failed to test vgchange --poll y'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to test vgchange --poll y'
Thu Aug 31 17:28:58 2023 - ERROR - failed to test vgchange --poll y
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 44'
+ message='oe_test_lvm2_vgchange_002.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 44'
Thu Aug 31 17:28:59 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 44
+ return 0
+ vgchange --refresh --autobackup y test
  Volume group "test" not found
  Cannot process volume group test
+ CHECK_RESULT 5 0 0 'failed to test vgchange --refresh --autobackup y'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='failed to test vgchange --refresh --autobackup y'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'failed to test vgchange --refresh --autobackup y'
+ LOG_ERROR 'failed to test vgchange --refresh --autobackup y'
+ message='failed to test vgchange --refresh --autobackup y'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to test vgchange --refresh --autobackup y'
Thu Aug 31 17:29:00 2023 - ERROR - failed to test vgchange --refresh --autobackup y
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 46'
+ message='oe_test_lvm2_vgchange_002.sh line 46'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 46'
Thu Aug 31 17:29:01 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 46
+ return 0
+ vgchange --refresh --sysinit test
  Volume group "test" not found
  Cannot process volume group test
+ CHECK_RESULT 5 0 0 'failed to test vgchange --refresh --sysinit'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='failed to test vgchange --refresh --sysinit'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'failed to test vgchange --refresh --sysinit'
+ LOG_ERROR 'failed to test vgchange --refresh --sysinit'
+ message='failed to test vgchange --refresh --sysinit'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to test vgchange --refresh --sysinit'
Thu Aug 31 17:29:01 2023 - ERROR - failed to test vgchange --refresh --sysinit
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 48'
+ message='oe_test_lvm2_vgchange_002.sh line 48'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 48'
Thu Aug 31 17:29:02 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 48
+ return 0
+ vgchange --refresh --reportformat basic test
  Volume group "test" not found
  Cannot process volume group test
+ CHECK_RESULT 5 0 0 'failed to test vgchange --refresh --reportformat basic'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='failed to test vgchange --refresh --reportformat basic'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'failed to test vgchange --refresh --reportformat basic'
+ LOG_ERROR 'failed to test vgchange --refresh --reportformat basic'
+ message='failed to test vgchange --refresh --reportformat basic'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to test vgchange --refresh --reportformat basic'
Thu Aug 31 17:29:03 2023 - ERROR - failed to test vgchange --refresh --reportformat basic
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 50'
+ message='oe_test_lvm2_vgchange_002.sh line 50'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 50'
Thu Aug 31 17:29:03 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 50
+ return 0
+ vgchange --refresh --reportformat json test
  Volume group "test" not found
  Cannot process volume group test
  {
  }
+ CHECK_RESULT 5 0 0 'vgchange --refresh --reportformat json'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='vgchange --refresh --reportformat json'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'vgchange --refresh --reportformat json'
+ LOG_ERROR 'vgchange --refresh --reportformat json'
+ message='vgchange --refresh --reportformat json'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'vgchange --refresh --reportformat json'
Thu Aug 31 17:29:04 2023 - ERROR - vgchange --refresh --reportformat json
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_vgchange_002.sh line 52'
+ message='oe_test_lvm2_vgchange_002.sh line 52'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_vgchange_002.sh line 52'
Thu Aug 31 17:29:05 2023 - ERROR - oe_test_lvm2_vgchange_002.sh line 52
+ return 0
+ LOG_INFO 'End executing testcase!'
+ message='End executing testcase!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End executing testcase!'
Thu Aug 31 17:29:05 2023 - INFO  - End executing testcase!
+ CASE_RESULT 0
+ case_re=0
+ test -z 9
+ test 9 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Aug 31 17:29:06 2023 - ERROR - failed to execute the case.
+ exit 9
+ post_test
+ LOG_INFO 'Start environment cleanup.'
+ message='Start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environment cleanup.'
Thu Aug 31 17:29:06 2023 - INFO  - Start environment cleanup.
+ vgremove -f test
  Volume group "test" not found
  Cannot process volume group test
+ pvremove -f /dev/
  No device found for /dev/.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Thu Aug 31 17:29:07 2023 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'Finish environment cleanup.'
+ message='Finish environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup.'
Thu Aug 31 17:29:08 2023 - INFO  - Finish environment cleanup.
